content/departments/product-engineering/engineering/cloud/delivery/on-call.md

```diff
@@ -0,0 +1,126 @@
+# Delivery Team On-Call Rotation
+
+The Delivery Team maintains an on-call rotation where an engineer is dedicated to providing support to our
+internal teams and managed instances for a term of 1 week.
+
+The on-call schedule is captured in OpsGenie. The on-call rotation consists of an Engineer On-Call (EOC), a Secondary Engineer On-Call (SEOC), and a technical manager (TM).
+
+On-Call duties transfer automatically Mondays at 10am Central Time according to the schedule within OpsGenie.
+
+## Response Times and SLAs
+
+The Delivery team aims to provide timely responses that are proportional to the capacity of the team.
+The Engineer On-Call (EOC) is responsible for a 24 hour x 7 period shift, guarenteeing responses during their
+working hours (generally 9am - 5pm in their timezone).
+
+The EOC is expected to provide an initial response within 45 minutes in response to an Incident where they are activated with an OpsGenie page. If the EOC is unavailable within the first 45 minutes of an OpsGenie page, the Secondary Engineer On-Call will be activated, where the expectation is to provide an initial response within an additional 45 minutes.
+
+## Roles
+
+### Incoming Engineer On-Call (IEOC)
+
+The engineer that is starting their on-call rotation.
+
+### Outgoing Engineer On-Call (OEOC)
+
+The engineer that is ending their on-call rotation.
+
+### Engineer On-Call (EOC)
+
+The engineer that is currently on-call.
+
+### Secondary Engineer On-Call (SEOC)
+
+The engineer that serves as an escalation if the Engineer On-Call is not available.
+
+### Technical Manager
+
+The manager of the Delivery team exists as the terminal escalation for the on-call rotation.
+
+---
+
+# Delivery Team On-Call Playbook
+
+## Starting On-Call Rotation
+
+At the start of their shift, the Incoming Engineer On-Call (IEOC) will prepare to take over on-call duties from the Outgoing Engineer On-Call.
+
+### Review Hand-off Notes
+
+The previous on-call engineer should post in the #delivery channel a link to the [Delivery On-Call Hand-off Google Doc](https://docs.google.com/document/d/1K1ZsfocwLk9F6Do4DZQImiSmh-ln2L3IomDBlxxdQHs/edit?usp=sharing)
```

Comment:
What do you think about using the delivery-scratch repo instead, so it's searchable? Or is the ease of using a google doc better?

Reply:
Love it! Gotta have it. I will have a PR against the scratch repo shortly with the template.